src/test/java/io/strimzi/kafka/quotas/StaticQuotaCallbackTest.java

```diff
@@ -180,7 +181,7 @@ void shouldScheduleStorageChecker() {
         target.configure(MINIMUM_EXECUTABLE_CONFIG);
 
         //Verify
-        verify(scheduledExecutorService, times(1)).scheduleWithFixedDelay(any(), eq(0L), eq(10L), eq(TimeUnit.SECONDS));
+        verify(scheduledExecutorService, times(2)).scheduleWithFixedDelay(any(), eq(0L), eq(10L), eq(TimeUnit.SECONDS));
```

Comment:
I wonder if we should make this assertion more discerning and expect a specific type of submission? 

Having said that it would be more specific anyway if we change the delay fallback is executed with.